tests/foreman/ui/test_ansible.py

```diff
@@ -602,6 +604,56 @@ def test_positive_ansible_config_report_changes_notice_and_failed_tasks_errors(
             assert f'notice Install the {package_name} package' in notice_log['permission_denied']
             assert f'Installed: rubygem-{package_name}' in notice_log['permission_denied']
 
+    @pytest.mark.no_containers
+    @pytest.mark.rhel_ver_list([settings.content_host.default_rhel_version])
+    def test_positive_debug_log_for_host(
```

Comment:
could you move this test to AnsibleREX?
```suggestion
    def test_positive_ansible_job_with_verbose_stdout(
```